src/components/common/HelpText.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <header id="name-header">
```

Comment:
Our convention is that the top element (which isn't template) holds the id of this component.

In this case, it would be "help-text", but that's very generic and you've put this component in the common folder. Please move it to common/Amalgamation folder and rename this more specifically in case we have other help text components.